src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
+const CAR_RENT_PER_DAY = 40;
+const SALE_FOR_WEEK_AND_MORE = 50;
+const SALE_FOR_SHORT_TERM = 20;
+const SEVEN_DAYS_RENT = 7;
+const THREE_DAYS_RENT = 3;
+
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_SUM = CAR_RENT_PER_DAY * days;
+  if (days >= SEVEN_DAYS_RENT) {
+    return TOTAL_SUM - SALE_FOR_WEEK_AND_MORE;
+  }
+
+  if (days >= THREE_DAYS_RENT) {
+    return TOTAL_SUM - SALE_FOR_SHORT_TERM;
+  }
+
+  return TOTAL_SUM;
```

Comment:
```suggestion
  const totalSum = CAR_RENT_PER_DAY * days;
  if (days >= SEVEN_DAYS_RENT) {
    return totalSum - SALE_FOR_WEEK_AND_MORE;
  }

  if (days >= THREE_DAYS_RENT) {
    return totalSum - SALE_FOR_SHORT_TERM;
  }

  return totalSum;
```

Reply:
In `UPPER_CASE` we name only variables that contain constant value(as all that are above 40, 50, 20....). This value is not constant, it's calculated, so we name it in `camelCase`